codebook.toml

```diff
@@ -20,6 +20,7 @@ words = [
     "efa",
     "eprintln",
     "errno",
+    "ethtool",
```

Comment:
How comes this didn't show up earlier? Are we not running the spell checker in CI? :thinking:

Reply:
I just happened to see that the spell check was flagging ethtool.  Does the spell check pose a problem?